include/blmc_robots/blmc_joint_module.hpp

```diff
@@ -124,6 +124,13 @@ class BlmcJointModule
      */
     void send_torque();
 
+    /**
+     * @brief Get the max joint torque that can be safely applied
```

Comment:
i would slightly change the formulation here, because this torque may not necessarily be safe, especially not for an extended time duration (it may burn the motor or lead to collisions etc.).
so i would just say something like maximum admissible torque

Reply:
Agreed